src/bench/bench.cpp

```diff
@@ -23,6 +23,9 @@ void
 benchmark::BenchRunner::RunAll(benchmark::duration elapsedTimeForOne)
 {
     perf_init();
+    if (std::ratio_less_equal<benchmark::clock::period, std::micro>::value) {
```

Comment:
I assume you meant to invert that condition, i.e. print the warning when our period is **not** less_equal micro-precision.